_scripts/build_person_index.py

```diff
@@ -41,12 +41,13 @@ def years_ago(earlier_date, later_date):
     else:
         return (later_date.year - earlier_date.year) - 1
 
-
+party_dict = {}
+for party in ynmp_export['organizations']:
+    party_dict[party['id']] = party
```

Comment:
This should probably check that the organisation classification is "Party" to be strictly correct, doesn't matter though since it's just building an index.